.github/workflows/ci.yaml

```diff
@@ -322,6 +322,51 @@ jobs:
           set -euxo pipefail
           mv linux/rust.require.cmake linux/rust.cmake
           flutter test -d linux integration_test/main.dart --verbose
+  
+  dart_web_test:
+    name: Dart (Web) unit tests
+    runs-on: ubuntu-latest
+    
+    steps:
+      - uses: actions/checkout@v2
+      - uses: dart-lang/setup-dart@v1
+      - name: Install Rust toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: nightly
+          target: wasm32-unknown-unknown
+          components: rust-src,rustfmt
+      - name: Install ffigen
+        run: dart pub global activate ffigen
+      - name: Cache dependencies
+        id: test-cache
+        uses: actions/cache@v3
+        with:
+          path: |
+            ~/.cargo/bin/
+            ~/.cargo/registry/index/
+            ~/.cargo/registry/cache/
+            ~/.cargo/git/db/
+            frb_codegen/target
+            frb_example/pure_dart/rust/target/
+            frb_example/pure_dart/dart/.local-chromium
+          key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
```

Comment:
Cache "dart chrome" by "rust cargo lock" - seems a problem?

Reply:
Shouldn't be an issue, other than that Rust deps upgrades will also invalidate the cached Chromium binary.